src/library_webgpu.js

```diff
@@ -1481,7 +1481,7 @@ var LibraryWebGPU = {
     assert(commands % 4 === 0);
 #endif
     var queue = WebGPU.mgrQueue.get(queueId);
-    var cmds = Array.from(HEAP32.subarray(commands >> 2, (commands >> 2) + commandCount),
+    var cmds = Array.from({{{ makeHEAPViewWithCount(`${POINTER_BITS}`, 'commands', 'commandCount')}}},
```

Comment:
Would you mind just using the existing `makeHEAPView` here and doing `commandCount*4` here at the call site.  This is how a lot of existing call sites work.. once this change lands we can look into creating a new version of `makeHEAPView` as a separate change perhaps?

Reply:
Done.